rfd/0107-abandon-active-access-request.md

```diff
@@ -0,0 +1,103 @@
+---
+authors: Jeff Pihach (jeff.pihach@goteleport.com)
+state: draft
+---
+
+# RFD 0107 - Abandon Active Access Requests
+
+## Required Approvers
+
+- Engineering: @zmb3
+
+## What
+
+Allow users who are using elevated permissions via an access request to abandon
+them ahead of their expiry once they are no longer of use.
```

Comment:
```suggestion
Allow users who are using elevated permissions via an access request to abandon
them ahead of their expiry once they are no longer of use. Once a request has been
abandoned, it can no longer be used to assume elevated privileges.
```